bootstrap.ps1

```diff
@@ -7,31 +7,22 @@
 #>
 
 param(
-  [string[]] $PackageDirs
+  [string] $PackageName,
+  [string] $CondaEnvironmentSuffix,
+  [bool] $FromSource
 )
 
-& (Join-Path $PSScriptRoot "build" "set-env.ps1");
+# Set env vars
+& (Join-Path (Join-Path $PSScriptRoot "build") "set-env.ps1");
 
-Import-Module (Join-Path $PSScriptRoot "build" "conda-utils.psm1");
+# Import Conda utils
+Import-Module (Join-Path (Join-Path $PSScriptRoot "build") "conda-utils.psm1");
 
 # Enable conda hook
 Enable-Conda
 
-# Get default value for PackageDirs by searching for environment.yml files
-if ($null -eq $PackageDirs) {
-  $PackageDirs = Get-ChildItem -Path $PSScriptRoot -Recurse -Filter "environment.yml" | Select-Object -ExpandProperty Directory | Split-Path -Leaf
-  Write-Host "##[info]No PackageDir. Setting to default '$PackageDirs'"
-}
+# Create environment
+& (Join-Path (Join-Path $PSScriptRoot build) create-env.ps1) -PackageName $PackageName -CondaEnvironmentSuffix $CondaEnvironmentSuffix
```

Comment:
would it make sense to move create-env.ps1 inside conda-utils.psm1?

Reply:
I created a new module 'package-utils.psm1' for this since the function takes a `PackageName`, so it is package-specific and not universal to Conda environments.